string[] bin_ids
string[] object_ids
string[] object_asins
string[] object_names
---
uint8[] outcomes
